packages/cedar-amcharts/src/render/render.ts

```diff
@@ -44,6 +44,7 @@ export function fillInSpec(spec: any, definition: any) {
 
   // Add a legend in case it's not on the spec
   if (!spec.legend) { spec.legend = {} }
+  if (definition.legend && definition.legend.hasOwnProperty('enable')) { definition.legend.visible = definition.legend.enable }
```

Comment:
pls add a comment here w/ a quick note explaining why this is needed and that it should be removed next breaking change.